GHzDACs/jumpTable.py

```diff
@@ -189,49 +241,102 @@ class JumpTable(object):
     TODO: make self.jumps a property with some checking on type and number
     """
     PACKET_LEN = 528
-    COUNT_MAX = 2**32 - 1 #32 bit register for counters
-    
-    def __init__(self, startAddr=None, jumps=None):
-        self._counters = [0,0,0,0]        
-        self.startAddr = startAddr
+    COUNT_MAX = 2 ** 32 - 1  # 32 bit register for counters
```

Comment:
I think exponentiation is the one operator where it looks nicer without the whitespace, but this is a seriously trivial nit.